### DISCUSSION TITLE
Responsive width not working

### DISCUSSION THREAD

**Discussion by Isabela-Ferrer** [twbs/bootstrap#39344] [Q&A]
Hi! 

I am trying to create a responsive width for a card group on my website and it is not working. When I add classes to my parent container like these: "w-sm-100 w-md-75" the smaller one always overrides the other widths and it ends up having a width of 100 in all screen sizes. Any ideas why this might be?

```html
<div class="container w-sm-100 w-md-75" id="card-group" >
        <div class="row row-cols-lg-2 row-cols-1" id="about-me">
            <div class="col mx-auto text-white d-flex flex-column">
                <div class="card h-100 p-5 my-4">
                    <h1 class="mb-4 display-5" >About Me</h1>
                    <p class="">Welcome to my website! <br> <br>
               The rest of my content
            </div>
        </div>
    </div>
 ```

**Comment by julien-deramond** [comment]
Hi @Isabela-Ferrer 
Responsive width utilities don't exist by default in Bootstrap (such as `w-sm-100` and `w-md-75`). You can create them by using the Utility API; we have a [Responsive example](https://getbootstrap.com/docs/5.3/utilities/api/#responsive) showing how to do that.
